src/macaron/config/defaults.ini

```diff
@@ -64,7 +64,6 @@ artifact_ignore_list =
 # Duplicated host names are ignored.
 allowed_hosts =
     github.com
-    ol-bitbucket.us.oracle.com
```

Comment:
I don't think it's `bitbucket.org` as this page is the product page, not the page for the git host itself.

Reply:
> I don't think it's `bitbucket.org` as this page is the product page, not the page for the git host itself.

I think it is though, e.g., see https://bitbucket.org/jensdietrich/webfuzzer-paper/

Reply:
Also created a ticket to add more documentation for these Git services: https://github.com/oracle-samples/macaron/issues/285

Reply:
> https://bitbucket.org/jensdietrich/webfuzzer-paper/

It returned 404 for me (propably because I don't have permission) :sweat_smile: . I will update it to `bitbucket.org`

Reply:
Hmmm changing it to `bitbucket.org` might break some of the existing tests (both unit tests and integration tests) we have because there are repositories in the SBOM files has a bitbucket link, which was not detected before (e.g in this file `https://raw.githubusercontent.com/oracle-samples/macaron/main/tests/dependency_analyzer/cyclonedx/resources/root_bom.json` there is `https://bitbucket.org/snakeyaml/snakeyaml/`).

Reply:
Then let's move it to another milestone. I prefer to fix it properly.

Reply:
I agree